src/Atmos/Model/AtmosModel.jl

```diff
@@ -174,10 +179,33 @@ struct AtmosModel{
     tracers::TR
     "Large-scale forcing (Forcing information from GCMs, reanalyses, or observations)"
     lsforcing::LF
+    "Compressibility switch"
+    compressibility::C
     "Data Configuration (Helper field for experiment configuration)"
     data_config::DC
 end
 
+abstract type Compressibilty end
+
+"""
+    Compressible <: Compressibilty
+
+Dispatch on compressible model (default)
+
+ - Density is prognostic
+"""
+struct Compressible <: Compressibilty end
+
+"""
+    Anelastic1D <: Compressibilty
+
+Dispatch on Anelastic1D model
+
+ - Density is constant in time
+ - Remove momentum z-component tendencies
```

Comment:
@charleskawczynski Are the z-components actually removed by default? The changes to eq_tends eliminate mass advection and the momentum pressure gradient, but the z-components are only removed in stable_bl_anelastic1d.jl. Either this comment is misleading, or that z-component removal should be done by default.

Reply:
We should remove it by default. I think we can do this with the projection hook. We'll probably want to change the `Σfluxes`/`Σsources` argument list (and not slurp everything).

Reply:
Ok, now it should be hooked in by default.